content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -2,180 +2,278 @@
 Product variants
 ================
 
-Product variants are used to give single products a variety of different characteristics, such as
-size and color. Products using variants can be managed at the product template level (for all
-attributes and variants of that product), and/or at the variant level (for individual variants).
+Product variants are used to give single products a variety of different characteristics/options for
+customers to choose from, such as size, style, or color -- just to name a few.
 
-As an example, a company selling t-shirts may have the following product:
+Products that have variants can be managed on the product's individual product template (for all
+attributes and variants of that product), and/or on the :guilabel:`Product Variants` or
+:guilabel:`Attributes` page (for specific variants). All of which are located in the Odoo *Sales*
+application.
```

Comment:
I fully understand what you're suggesting, and I can see the benefit of such an explanation. However, since the example I use immediately follows the brief intro, I feel like adding a couple of sentences right before the example may come across as redundant. I believe the broken down example of a T-Shirt company does a better job of defining the differences between those elements. But, if you think it's necessary, I can reconsider and put a sentence or two in before the example. :)